### PR TITLE
request: preserver request order.

### PR DESCRIPTION
**Pull request by zannkukai**
On the item detailed view, you can display requests made by user about
this item. The request list should be sorted by creation date but
wasn't. This commit corrects this problem.

Closes rero/rero-ils#1314

Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Dependencies

https://github.com/rero/rero-ils/pull/1344

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html [hunk 1]
```diff
@@ -47,23 +47,25 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
         {{ 'Type' | translate }}:
       </dt>
       <dd class="col-sm-7 col-md-8 mb-0">
-        {{ record.metadata.item_type.pid | getRecord:'item_types':'field':'name' | async }}
+        {{ record.metadata.item_type.pid | getRecord:'item_types':null:'name' | async }}
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:50]: Why `null` ?

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html [hunk 2]
```diff
@@ -47,23 +47,25 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
...
       </dd>
       <!-- DOCUMENT -->
       <dt class="col-sm-3 offset-sm-2 offset-md-0">
         {{ 'Document' | translate }}:
       </dt>
       <dd class="col-sm-7 col-md-8 mb-0">
         <a [routerLink]="['/records', 'documents', 'detail', record.metadata.document.pid]">
-          {{ record.metadata.document.pid | getRecord:'documents':'field':'title' | async | mainTitle }}
+          {{ record.metadata.document.pid | getRecord:'documents':null :'title' | async | mainTitle }}
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:58]: Why `null` ?

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html [hunk 3]
```diff
@@ -47,23 +47,25 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
...
         </a>
       </dd>
       <!-- LIBRARY -->
       <dt class="col-sm-3 offset-sm-2 offset-md-0">
         {{ 'Library' | translate }}:
       </dt>
       <dd class="col-sm-7 col-md-8 mb-0">
-        <ng-container *ngIf="library">{{ library.metadata.name }}</ng-container>
+        <ng-container *ngIf="location">
+          {{ location.metadata.library.pid | getRecord: 'libraries': null: 'name' | async}}
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:67]: Why `null` ?
Missing space after `async`

File: projects/admin/src/app/record/detail-view/item-detail-view/item-transactions/item-transactions.component.html [hunk 4]
```diff
@@ -14,59 +14,46 @@
...
-        <ng-container *ngIf="requestedBy.length <= 1; else requests">
-          <b>{{ 'Request' | translate }}</b>
-        </ng-container>
-        <ng-template #requests>
-          <b>{{ 'Requests' | translate }}</b>
-        </ng-template>
-        <b> ({{ requestedBy.length }})</b>
-        <button class="btn btn-outline-primary btn-link btn-sm ml-2 float-right" (click)="addRequest(item.metadata.pid)">
-          <i class="fa fa-plus-square-o"></i> {{ 'Add' | translate }} ...
+        <b>{{ requestedBy.length | i18nPlural: {'=1': 'Request', 'other': 'Requests'} | translate }}</b>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-transactions/item-transactions.component.html:37]: `{ '=1': 'Request' | translate , 'other': 'Requests' | translate }` ?

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message proposition:

```
item: keep request order

The item detailed view displays the requests made by users. The request
list should be sorted by creation date but isn't. This commit fix this
issue.

Closes rero/rero-ils#1314
```

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html (hunk 1 above)
Comment (zannkukai) [reply projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:50]: oups

**Review by Garfield-fr — APPROVED** [review]

File: projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html [hunk 5]
```diff
@@ -47,23 +47,25 @@ <h1>{{ 'Barcode' | translate }} {{ record.metadata.barcode }}</h1>
...
+          {{ record.metadata.document.pid | getRecord:'documents': 'field' :'title' | async | mainTitle }}
         </a>
       </dd>
       <!-- LIBRARY -->
       <dt class="col-sm-3 offset-sm-2 offset-md-0">
         {{ 'Library' | translate }}:
       </dt>
       <dd class="col-sm-7 col-md-8 mb-0">
-        <ng-container *ngIf="library">{{ library.metadata.name }}</ng-container>
+        <ng-container *ngIf="location">
```
Comment (AoNoOokami) [inline projects/admin/src/app/record/detail-view/item-detail-view/item-detail-view.component.html:66]: Is this change described in the commit message?

**Review by AoNoOokami — CHANGES_REQUESTED** [review]

**Review by AoNoOokami — APPROVED** [review]

**Pull request merged**